pango/Gir.toml

```diff
@@ -282,6 +281,11 @@ status = "generate"
         name = "language"
         const = true
 
+[[object]]
+name = "Pango.GlyphItem"
```

Comment:
This has a `PangoItem`, which has a `PangoAnalysis`, which has a `PangoEngineShape`, `PangoEngineLang`, `PangoFont` and `PangoLanguage`. Are these all OK (language seems so, I didn't check the others)? If so you probably want to mark them here too, otherwise this needs some explanation why it is OK :)

Reply:
Good catch, `PangoFont` isn't safe. Removed this one